sw/device/rom_exts/rom_ext_start.S

```diff
@@ -92,7 +92,9 @@ _rom_ext_interrupt_vector:
   // On Ibex interrupt id 31 is for non-maskable interrupts.
   unimp
 
-  // Ibex Reset Handler:
+  // ROM_EXT entry point. ROM_EXT is the second boot stage, which means that
```

Comment:
minor nit: This file uses two different comment styles.

Reply:
It does. Would you prefer a `/* */` style comments for every multiline comment?

Reply:
I think my original intention was to differentiate section-like headers and exposition using `/* */`, and then use `//` for implementation-level comments, i.e. to differentiate what this code must do vs from what it does do. 

Given this wasn't so obvious, I'm happy to standardise on one comment style.

Reply:
> It does. Would you prefer a /* */ style comments for every multiline comment?

Yes, I think it would be better, thanks!

Reply:
Cool, will fix it up.

Reply:
Done.